.github/workflows/test.yml

```
@@ -0,0 +1,30 @@
+name: Test
+
+on:
+  push:
+    branches-ignore:
+      - main # tests will be launched by workflow_call from the "Release" workflow
+  pull_request:
+    types: [ opened, reopened ]
+  workflow_call:
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        operating_system: 
+          - ubuntu-20.04
```

Since this module does not depend on MongoDB, we can test on Ubuntu 22 🙂 (see https://github.com/OpenTermsArchive/engine/pull/974)
```suggestion
          - ubuntu-latest
```